tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -23,6 +24,28 @@
     sentry_files = json.load(fd)
 
 
+UNSUPPORTED_FRAME_FILENAMES = [
```

Comment:
Moving everything we do not support into here so we can easily inspect it. It also helps testing the same structure in the bucketing logic.